pyo3-macros-backend/src/module.rs

```diff
@@ -83,10 +86,60 @@ pub fn pymodule_impl(
             unsafe { #module_def_name.module_init() }
         }
 
+        #[cfg(any(not(feature = "pep489"), PyPy))]
         #[doc(hidden)]
         #visibility static #module_def_name: #krate::impl_::pymodule::ModuleDef = unsafe {
             #krate::impl_::pymodule::ModuleDef::new(concat!(stringify!(#name), "\0"), #doc, #krate::impl_::pymodule::ModuleInitializer(#fnname))
         };
+
+        #[cfg(all(feature = "pep489", not(PyPy)))]
```

Comment:
Hmm, looks like after macro expansion the code with this `cfg` are disappeared... 🧐